outis-core/src/main/scala/io/saagie/outis/core/anonymize/Anonymize.scala

```diff
@@ -0,0 +1,34 @@
+package io.saagie.outis.core.anonymize
+
+import scala.util.Random
+
+object Anonymize {
+
+  val letters = ('a' to 'z') ++ ('A' to 'Z')
+  val random = Random
+
+  def suppression(): String = {
+      null
+  }
+
+  def setTo(value: String, pattern: Char = ' '): String = value.map(_ => pattern)
+
+  def setToBlank(value: String): String = setTo(value)
+
+  def setToX(value: String): String =  setTo(value, 'X')
+
+  def truncate(value: String, size: Int): Either[AnonymizationException, String] = {
+    if (value.length > size)
+      return Right(value.substring(0, size))
```

Comment:
return could be removed and Left/Right and result value will be the value of the if/else statement